lib/discourse_automation/scripts/llm_triage.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+DiscourseAutomation::Scriptable::LLM_TRIAGE = "llm_triage"
+
+DiscourseAutomation::Scriptable.add(DiscourseAutomation::Scriptable::LLM_TRIAGE) do
+  version 1
+
+  placeholder :creator_username
+
+  triggerables %i[post_created_edited]
+
+  script { |context, fields, automation| }
+end
```

Comment:
Maybe it's just here temporary but you can add it to discourse-ai plugin like this:

```ruby
  if defined?(DiscourseAutomation)
    add_automation_scriptable("llm_triage") do
      placeholder :creator_username

      triggerables %i[post_created_edited]

      script do |context, fields, automation|
   
      end
    end
  end
```

Then you will need locales like this:

```yml
en:
  js:
    discourse_automation:
      scriptables:
        llm_triage:
           ...
```